packages/tasks/src/snippets/python.ts

```diff
@@ -432,15 +433,16 @@ export function getPythonInferenceSnippet(
 	model: ModelDataMinimal,
 	accessToken: string,
 	provider: SnippetInferenceProvider,
+	providerModelId?: string,
 	opts?: Record<string, unknown>
 ): InferenceSnippet[] {
 	if (model.tags.includes("conversational")) {
 		// Conversational model detected, so we display a code snippet that features the Messages API
-		return snippetConversational(model, accessToken, provider, opts);
+		return snippetConversational(model, accessToken, provider, providerModelId, opts);
```

Comment:
in theory given `model` and `provider`, then `providerModelId` is unneeded/redundant, because `model` contains the provider mapping.

But in practice it's probably simpler to do it explicitly like that